### PR TITLE
TA#19837 Payroll preparation export wizard

### PR DESCRIPTION
**Pull request by ddufresne**

### CODE REVIEW

File: payroll_preparation_export_wizard/tests/test_generate_file.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import base64
+import pytest
```
Comment (ddufresne) [inline payroll_preparation_export_wizard/tests/test_generate_file.py]: Codacy Issue found: [Unused import pytest](https://app.codacy.com/manual/numigi/odoo-timesheet-addons/pullRequest?prid=5136851)

File: payroll_preparation_export_wizard/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
```
Comment (ddufresne) [inline payroll_preparation_export_wizard/__manifest__.py]: Codacy Issue found: [Statement seems to have no effect](https://app.codacy.com/manual/numigi/odoo-timesheet-addons/pullRequest?prid=5136851)

File: payroll_preparation_export_wizard/wizard/payroll_preparation_export.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import base64
+from odoo import fields, models, _
+from odoo.exceptions import AccessError
```
Comment (ddufresne) [inline payroll_preparation_export_wizard/wizard/payroll_preparation_export.py]: Codacy Issue found: [Unused AccessError imported from odoo.exceptions](https://app.codacy.com/manual/numigi/odoo-timesheet-addons/pullRequest?prid=5136851)

**Review by foutoucour — CHANGES_REQUESTED** [review]
Il y a des folders __pycache__ embarqué. Il faudrait mettre à jour le .gitignore

**Comment by ddufresne** [comment]
@foutoucour done

**Review by foutoucour — APPROVED** [review]

**Pull request merged**